2-algorithms/3-python/Shellsort.MD

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
Add h1 heading

Reply:
your content seems good, but we need the definition, simple explantion and the code level explanation. please refer this MD file and make changes accordingly https://github.com/Abishethvarman/programming-resources/blob/main/2-algorithms/1-java/merge-sort.MD